src/LibSassBuilder/FilesOptions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CommandLine;
+using System.Collections.Generic;
+
+namespace LibSassBuilder
+{
+	[Verb("files")]
+	public class FilesOptions : GenericOptions
+	{
+		[Value(0, Required = true, HelpText = "Directory in which to run. Defaults to current directory.")]
```

Comment:
Could you please give a bit more descriptive `HelpText` here.

Reply:
haha, that was your original text ;-)

Reply:
Sry, I'll check... got my view of your comment a little messed up ;-)

Reply:
Done